packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -96,6 +96,19 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
         : XAxisTypes.StringAxis;
   }
 
+  public componentDidMount(): void {
+    const isChartEmpty = d3Min(this._points, (point: IVerticalBarChartDataPoint) => point.y) === 0;
+    if (isChartEmpty) {
+      d3Select('body')
+        .append('div')
+        .attr('role', 'alert')
+        .attr('id', 'ariaLabel_VerticalBarChart')
+        .style('opacity', 0)
+        .attr('aria-label', 'Graph has no data to display')
+        .attr('tabIndex', 0);
+    }
+  }
+
   public render(): JSX.Element {
```

Comment:
Can you make this change for other graphs also when they are rendered as empty.

Reply:
sure, creating a separate PR for rest of the graphs.